### DISCUSSION TITLE
No strelka logs in kibana

### DISCUSSION THREAD

**Discussion by Mastadamus** [Security-Onion-Solutions/securityonion#6677] [General] [answered]
I am running security onion 2.3.90 based on the security onion ISO with multiple iterations of soup to get it to 2.3.90 started at 2.3.50 I believe. 
I recently realized I am not getting strelka logs ingested into kibana.  A check on the indices shows no documents in elastic strelka indice. 
Everything is green for so-status. Additionally, when I checked the output of the strelka.log nothing popped up. I've tried restarting strelka to no avail.  nothing in logstash logs that mention strelka. 
![image](https://user-images.githubusercontent.com/79027361/147175358-4ae61a55-bf0a-40ac-a28c-2a884a403a48.png)
![image](https://user-images.githubusercontent.com/79027361/147175396-8f30c9df-f119-4b0a-8e9f-5829e1f70d04.png)
![image](https://user-images.githubusercontent.com/79027361/147175447-013448f2-5650-4e93-bb44-c0129b99bb75.png)

Any suggestions? help would be greatly appreciated.

**Comment by dougburks** [comment, marked as answer]
Are you sure you are seeing network traffic with files transferred in cleartext?

Do you have anything in /nsm/strelka/processed/?

**Reply by Mastadamus** [reply]
Yes sir. I have a decent amount of files in that directory and I even downloaded some pdf files from an http source in the clear just to be certain.

**Reply by Mastadamus** [reply]
I did notice this when I ran salt highstate

``` ----------
          ID: strelkaportavailable
    Function: cmd.run
        Name: netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0
      Result: True
     Comment: Command "netstat -utanp | grep ":57314" | grep -qvE 'docker|TIME_WAIT' && PROCESS=$(netstat -utanp | grep ":57314" | uniq) && echo "Another process ($PROCESS) appears to be using port 57314.  Please terminate this process, or reboot to ensure a clean state so that Strelka can start properly." && exit 1 || exit 0" run
     Started: 03:38:55.014416
    Duration: 68.872 ms
     Changes:
              ----------
              pid:
                  5652
              retcode:
                  0
              stderr:
              stdout:
 Im not sure if this is applicable. I didn't get any failures in the salt highstate though.

**Reply by Mastadamus** [reply]
I fixed this. I re pulled all the docker containers then rebooted and I started getting strelka logs in kibana.
